app/assets/javascripts/bootstrap-datepicker/v1.6.4/locales/bootstrap-datepicker.kh.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
@hengsokly: .kh is incorrect and will be updated to .km in bootstrap 1.7.0, https://github.com/eternicode/bootstrap-datepicker/blob/master/js/locales/bootstrap-datepicker.km.js.

Don't forget to update :)